pkg/volume/csi/csi_attacher.go

```diff
@@ -197,8 +201,14 @@ func (c *csiAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.No
 		}
 
 		attachID := getAttachmentName(volumeHandle, driverName, string(nodeName))
+		attach, err := c.plugin.volumeAttachmentLister.Get(attachID)
+		if (err == nil) {
```

Comment:
nit: we could avoid `( ..)` around this check :)

Reply:
Done.